### ISSUE TITLE
[Feature Request] Drag items with right-click

### ISSUE THREAD

**Issue by TheYellowArchitect** [peter-kish/gloot#242]
In the game  I'm making (think topdown shooter like Synthetik), left click shoots a gun, and also if you left click an item (to drag it), the item is activated. To move items to other inventory slots, right click is required. I don't know how easy that is to implement.

**Comment by peter-kish** [comment]
Yeah, that solution looks like a hack and Godot doesn't seem to "officially" support right-click drags ☹️
Since I'm trying to stick to the "Godot way" of implementing things, I don't see this being implemented soon (I had bad experience with including such solutions into my code).

But I'll leave the issue open in case something changes and Godot 4.4 starts supporting this feature.

**Comment by TheYellowArchitect** [comment]
I understand and agree. I would suggest to change the "wontfix" tag to "godot-core-engine" or sth, so its more explicit what holds an issue (and also apply it here)

Btw, aside of the above hack, how would you suggest to implement this right-click to drag feature?

**Comment by TheYellowArchitect** [comment]
>But I'll leave the issue open in case something changes and Godot 4.4 starts supporting this feature.

There is currently [no godot proposal for this ](https://github.com/godotengine/godot-proposals/issues?page=3&q=is%3Aissue+is%3Aopen+right+click), so it won't be implemented at 4.4 or later.
I would like to open it myself but you know far more than me, to the point you could even suggest how to implement left/right/middle mouse dragging in the code

If you don't want to, it's alright, I will open a godot proposal and try my best to explain the problem at least

**Comment by TheYellowArchitect** [comment]
I opened a godot proposal https://github.com/godotengine/godot-proposals/issues/10837
Whenever it is completed, this issue can close :+1:
